src/aqua/chain/new_deals.aqua

```diff
@@ -74,6 +77,10 @@ func join_deals(chain: ChainInfo, spell_id: SpellId, logs: []DealMatched, left:
                 -- This way, if particle has timed out during process of a log, `last seen block` would not be updated
                 -- And we will try all events from that block again
                 LastSeen.gt_set(spell_id, prev_block!, left)
+    -- Check deal statuses 
+    if deal_ids != nil:
```

Comment:
What will happen if TTL hits here?

Would these deal statuses be polled anyway? left boundary will be already moved, so logs won't come.

Reply:
Ah, I see, they will be polled for Joined Deals. I guess that detail deserves a comment here.

Reply:
Yeah, on the next run, we poll statutes for all joined deals

Reply:
is it only on the next run, or on the same run, too?

Reply:
if ttl happens, the all joined deals statuses will be polled on the next run, because there's no current run.
for newly joined deals we poll statuses _only_ after installation. 

In one run we sent two `getStatus` requests: one for newly joined deals, and the second for already joined deals. Maybe, it's better to send everything in one request, but it would require some refactoring of the decider (which I want to do both)